curriculum/challenges/english/25-front-end-development/workshop-sentence-analyzer/66e2edb23013ca596f8d9568.md

```diff
@@ -0,0 +1,85 @@
+---
+id: 66e2edb23013ca596f8d9568
+title: Step 6
+challengeType: 1
+dashedName: step-6
+---
+
+# --description--
+
+To start testing your `getPunctuationCount` function, create a `punctuationCount` variable and set it to the calling of `getPunctuationCount` function with a sentence, for example, `"WHAT?!?!?!?!?"`.
+
+After that, log the `punctuationCount` variable to the console with the text `"Punctuation Count:"` and the interpolation of the `punctuationCount` variable. You can also use plus (`+`) concatenator if you want.
+
+# --hints--
+
+You should create a `punctuationCount` variable.
+
+```js
+assert.isNotNull(punctuationCount)
+```
+
+Your `punctuationCount` variable should be set to the result of `getPunctuationCount("WHAT?!?!?!?!?")`.
+
+```js
+assert.equal(punctuationCount, getPunctuationCount("WHAT?!?!?!?!?"));
+```
+
+You should log the `punctuationCount` variable to the console with the parameter `"Punctuation Count: ${punctuationCount}"`.
```

Comment:
this hint of `"Punctuation Count: ${punctuationCount}"`  implies that we are only going to accept template literals here

```suggestion
You should log the following to the console: `"Punctuation Count: [Punctuation count goes here]"`. Replace  `[Punctuation count goes here]` with the the actual variable name. Make sure to use proper string concatenation syntax here.
```